contracts/IAOLinearVesting.sol

```diff
@@ -196,7 +218,8 @@ contract IAOLinearVesting is ReentrancyGuard, Initializable {
     /// @notice Calculate a users allocation based on the total amount deposited. This is done
     ///  by first scaling the deposited amount and dividing by the total amount.
     /// @param _user Address of the user allocation to look up
-    function getUserAllocation(address _user) public view returns (uint256) {
+    /// @notice This function has been deprecated, but leaving in the contract for backwards compatibility.
+    function getUserAllocation(address _user) external view returns (uint256) {
```

Comment:
Is the expectation for the frontend to do the Math? 

What is the proposed alternative?

Reply:
I believe getUserAllocation function is not deprecated here. It's just hinted for the external usage, if I'm not mistaken.

Reply:
Maybe `deprecated` is the wrong term, but it's deprecated in the sense that the contract no longer relies on its math.